__tests__/app.test.js

```diff
@@ -113,20 +113,45 @@ describe("getArticles", () => {
         });
       });
   });
-  test('404: should return not found', () => {
+  test("404: should return not found", () => {
     return request(app)
       .get("/api/articles/99/comments")
       .expect(404)
-      .then(({ body }) => { 
-        expect(body.msg).toBe(`ERROR: no article with that id found`)
-      })
+      .then(({ body }) => {
+        expect(body.msg).toBe(`ERROR: no article with that id found`);
+      });
   });
-  test('200: should be ordered by time created', () => {
+  test("200: should be ordered by time created", () => {
     return request(app)
       .get("/api/articles/1/comments")
       .expect(200)
-      .then(({ body }) => { 
-        expect(body.comments).toBeSortedBy(`created_at`, {descending: true})
-      })
+      .then(({ body }) => {
+        expect(body.comments).toBeSortedBy(`created_at`, { descending: true });
+      });
+  });
+});
+
+describe.only("post article comments", () => {

```

Comment:
Remember to remove .only before pushing - otherwise we cant be sure your code works and may miss something small, meaning that you could get issues later down the line/in frontend